dev/WebView2/InteractionTests/WebView2Tests.cs

```diff
@@ -155,27 +139,28 @@ private static int GetInstalledBrowserVersion()
             return browserBuildVersion;
         }
 
-        private static int GetLoaderBuildVersion()
+        private static int GetSdkBuildVersion()
         {
-            int loaderBuildVersion = 0;
-            string loaderPath = Path.Combine(Environment.CurrentDirectory, NativeMethods.LoaderName);
+            int sdkBuildVersion = 0;
+            string dllName = "Microsoft.Web.WebView2.Core.dll";
+            string dllPath = Path.Combine(Environment.CurrentDirectory, dllName);
 
             try
             {
-                FileVersionInfo loaderVersionInfo = FileVersionInfo.GetVersionInfo(loaderPath);
-                loaderBuildVersion = loaderVersionInfo.ProductBuildPart;
+                FileVersionInfo sdkVersionInfo = FileVersionInfo.GetVersionInfo(dllPath);
+                sdkBuildVersion = sdkVersionInfo.ProductBuildPart;
                 Log.Comment("WebView2Tests Init: Found {0}: version {1} [build version: {2}] (file: {3})",
-                            NativeMethods.LoaderName,
-                            loaderVersionInfo.ProductVersion,
-                            loaderBuildVersion,
-                            loaderVersionInfo.FileName
+                            dllName,
+                            sdkVersionInfo.ProductVersion,
+                            dllPath,
```

Comment:
> dllPath, [](http://example.com/codeflow?start=27&length=9)

Should be build version to match the string output

Reply:
Updated